### PR TITLE
wording in intersections

### PR DESCRIPTION
**Pull request by dlawin**
## What are you changing in this pull request and why?

I noticed the wording here is weird

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/reference/node-selection/set-operators.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Run snowplow_sessions, all ancestors of snowplow_sessions, fct_orders, and all a
 
 ### Intersections
 
-If multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),
+If multiple arguments to `--select`, `--exclude`, and `--selector` are comma-separated (with no whitespace in between),
```
Comment (dlawin) [inline website/docs/reference/node-selection/set-operators.md]:
```suggestion
Multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),
```
alternatively

File: website/docs/reference/node-selection/set-operators.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Run snowplow_sessions, all ancestors of snowplow_sessions, fct_orders, and all a
 
 ### Intersections
 
-If multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),
+Multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),
 dbt will select only resources which satisfy _all_ arguments.
```
Comment (mirnawong1) [inline website/docs/reference/node-selection/set-operators.md]:
```suggestion

```

File: website/docs/reference/node-selection/set-operators.md [hunk 3]
```diff
@@ -16,8 +16,7 @@ Run snowplow_sessions, all ancestors of snowplow_sessions, fct_orders, and all a
 
 ### Intersections
 
-If multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),
-dbt will select only resources which satisfy _all_ arguments.
+Multiple arguments to `--select`, `--exclude`, and `--selector` can be comma-separated (with no whitespace in between),
```
Comment (mirnawong1) [inline website/docs/reference/node-selection/set-operators.md]: great suggestion! i made a small tweak to clarify the language further!

```suggestion
If you separate multiple arguments for `--select`, `--exclude`, and `--selector` with commas and no whitespace in between, dbt will select only resources that satisfy _all_ arguments.
```

**Review by mirnawong1 — APPROVED** [review]
hey @dlawin great suggestion and thank you for opening up this pr to improve the docs! you're totally right and i made an add'l tweak to clarify the language as well. 

thanks again and looking forward to your future contributions ✨

**Pull request merged**